lib/nacl.js

```diff
@@ -23,7 +23,7 @@ function config(config, response) {
   opt.response = response;
   opt.baseUrl = options.baseUrl;
   opt.decodedObjectName = options.decodedObjectName;
-  opt.defautRole = options.defautRole || 'guest';
+  opt.defaultRole = options.defautRole || 'guest';
```

Comment:
Nice catch here, can you also change `options.defautRole` to `options.defaultRole`